### DISCUSSION TITLE
Hey i deploy my contract it give me error  and i'll match with the script file both are same if someone have idea how i slove this please tell me

### DISCUSSION THREAD

**Discussion by viveknath13** [Cyfrin/foundry-full-course-cu#2284] [Q&A]
foundry-NFT git:(master) ✗ make mint        
[⠒] Compiling...
No files changed, compilation skipped
Traces:
  [938879] → new mintBasicNft@0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496
    └─ ← [Return] 4579 bytes of code

  [27000] mintBasicNft::run()
    ├─ [0] VM::readDir("./broadcast", 3) [staticcall]
    │   └─ ← [Return] [("IO error for operation on /home/zenos/Documents/foundry-NFT/broadcast: No such file or directory (os error 2)", "/home/zenos/Documents/foundry-NFT/broadcast", 0, false, false)]
    ├─ [0] VM::toString(31337 [3.133e4]) [staticcall]
    │   └─ ← [Return] "31337"
    └─ ← [Revert] revert: No deployment artifacts were found for specified chain

Gas used: 48064
Error: 
script failed: revert: No deployment artifacts were found for specified chain
make: *** [makefile:8: mint] Error 1

**Comment by EngrPips** [comment]
Hello @viveknath13, Please read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2225#discussioncomment-10308553 . I believe it will help you resolve this issue, as this person ran into this exact same issue and resolved it
